lib/db/mysql.js

```diff
@@ -155,7 +156,7 @@ module.exports = function (log, error) {
 
   // Insert : accounts
   // Values : uid = $1, normalizedEmail = $2, email = $3, emailCode = $4, emailVerified = $5, kA = $6, wrapWrapKb = $7, authSalt = $8, verifierVersion = $9, verifyHash = $10, verifierSetAt = $11, createdAt = $12, locale = $13
```

Comment:
Although I'm dubious whether these comments add any value, as long as we have them we should be keeping them up to date I guess. This one may also be a smidgen overdue for a line-break I reckon. :smile: